include/functions.h

```diff
@@ -73,7 +73,11 @@ void player_input_to_move_vector(f32* angle, f32* magnitude);
 void game_input_to_move_vector(f32* x, f32* y);
 void exec_ShakeCamX(s32 arg0, s32 arg1, s32 arg2, f32 arg3);
 void exec_ShakeCam1(s32 arg0, s32 arg1, s32 arg2);
-f32 func_800E5348(void);
+
+// returns the angle that the player *sprite* is facing in world-space, this is NOT the direction the player character
```

Comment:
Probably worth specifying that this returns the target direction of the sprite, i.e. whilst turning around it interpolate

Reply:
It's not. The target sprite direction is into or out of the screen, while this is to the left or right of the player depending on which direction they are facing. This function is used for hammer and parasol collision checks, not to find the target for interpolation. Attached is a picture demonstrating this. I am walking toward the camera (yaw = 180), but facing left (yaw 270).
![forward](https://github.com/user-attachments/assets/1fa9841e-cf4a-48a7-97ea-bdbfbf7d1284)

Reply:
My comment was meant to be phrased as a question, but I failed at that somehow.. Thanks